deluge/plugins/Scheduler/deluge/plugins/scheduler/gtkui.py

```diff
@@ -13,26 +13,30 @@
 
 import logging
 
-import gtk
+from gi.repository import Gtk, Gdk
 
 import deluge.component as component
 from deluge.plugins.pluginbase import GtkPluginBase
 from deluge.ui.client import client
+from gi import pygtkcompat
```

Comment:
I think it is now `import pygtkcompat` (see https://twistedmatrix.com/trac/ticket/6280) but is it still needed in this plugin?